### PR TITLE
Update nest alert instructions for 2022.7

### PR DESCRIPTION
**Pull request by allenporter**
Update nest alert instructions for 2022.7 to point to new steps in documentation and interactive UI flow.

Related issues:
https://github.com/home-assistant/core/issues/74552
https://github.com/home-assistant/core/issues/67662

### CODE REVIEW

File: alerts/nest.markdown [hunk 1]
```diff
@@ -17,9 +17,11 @@ New users may sign up using *Web Auth* without issue. Follow the [documentation]
 
 ## Existing Users: App Auth
 
-If you created *App Auth* credentials after Feb 28, 2022 and your integration is currently broken, then follow the workaround instructions to move to *Web Auth* in the [github issue](https://github.com/home-assistant/core/issues/67662#issuecomment-1144425848).
+If you previously successfully configured Nest and Home Assistant with *App Auth* then follow the instructions for [Deprecated App Auth Credentials](https://www.home-assistant.io/integrations/nest/#deprecated-app-auth-credentials).
 
-For everyone else, a future version of Home Assistant will walk you through the steps to create new *Web Auth* credentials and upgrade your integration. You will need to upgrade before October to avoid interruption.
+Nest is now configured entirely from the UI using [Application Credentials](https://www.home-assistant.io/integrations/application_credentials/) and the configuration flow will walk you through the steps of creating new credentails the right away.
```
Comment (MartinHjelmare) [inline alerts/nest.markdown:22]:
```suggestion
Nest is now configured entirely from the UI using [Application Credentials](https://www.home-assistant.io/integrations/application_credentials/) and the configuration flow will walk you through the steps of creating new credentails the right way.
```

File: alerts/nest.markdown [hunk 2]
```diff
@@ -17,9 +17,11 @@ New users may sign up using *Web Auth* without issue. Follow the [documentation]
...
+
+You will need to upgrade before October to avoid interruption.
```
Comment (MartinHjelmare) [inline alerts/nest.markdown:24]: Maybe say explicitly in this section what needs to be upgraded?

Comment (MartinHjelmare) [reply alerts/nest.markdown:24]:
```suggestion
You will need to upgrade to *Web Auth* before October to avoid interruption.
```

**Review by frenck — APPROVED** [review]
Thanks, @allenporter 👍

**Pull request merged**